internal/filesystem.go

```diff
@@ -284,6 +284,9 @@ func FileSha256(filepath string) (sum []byte, err error) {
 // checks if command is in PATH
 func IsCommandAvailable(command string) bool {
 	_, err := exec.LookPath(command)
+	if err != nil {
+		log.Println(WarningPrefix, "command:", command, " is-available check result:", err)
```

Comment:
shouldn't log message say that the command is not available?

Reply:
To be removed. See last comment to @piotrjurkiewicz feedback.